php/FlatbufferBuilder.php

```diff
@@ -593,6 +593,14 @@ public function endVector()
 
     protected function is_utf8($bytes)
     {
+        if (function_exists('mb_detect_encoding')) {
+            if (!mb_detect_encoding($bytes, 'UTF-8', true)) {
```

Comment:
Because it's a non-default extension: http://php.net/manual/en/mbstring.installation.php